config/config.exs

```diff
@@ -70,11 +70,11 @@ config :godwoken_explorer, Oban,
     Oban.Plugins.Pruner,
     {Oban.Plugins.Cron,
      crontab: [
-       #  {"01 00 * * *", GodwokenIndexer.Worker.RefreshUDTSupply},
        {"10 00 * * *", GodwokenIndexer.Worker.DailyStat, args: %{datetime: nil}},
        {"*/2 * * * *", GodwokenIndexer.Worker.CheckLostAccount},
        {"*/10 * * * *", GodwokenIndexer.Worker.CheckContractCode},
-       {"* */4 * * *", GodwokenIndexer.Worker.RefreshNativeUDTSupply}
+       {"*/30 * * * *", GodwokenIndexer.Worker.RefreshBridgedUDTSupply},
+       {"* */1 * * *", GodwokenIndexer.Worker.UDTUpdater}
```

Comment:
Please add the reason for updating `bridged udt total supply` and `udt total supply` with different intervals.

Reply:
These two jobs have no connection. So the interval is free to set.